lib/active_admin/views/index_as_table.rb

```diff
@@ -213,7 +213,7 @@ def build(page_presenter, collection)
         table_options = {
           id: "index_table_#{active_admin_config.resource_name.plural}",
           sortable: true,
-          class: "index_table index",
+          class: "index-table js-index-table",
```

Comment:
I haven't heard of this style. What other projects do this?

I normally read the commits, and myself write detailed commit messages. I didn't read the commits this time simply because there's so many.